app/static/js/controllers/templateController.js

```diff
@@ -184,6 +168,20 @@ angular.module('uluwatuControllers').controller('templateController', ['$scope',
                 $scope.modifyStatusClass("has-error");
             });
         }
+        
+        function handleTemplateError(error, templateFailedMsg) {
+            var errorData = error.data["validationErrors"]
+            var errorMsg = $rootScope.error_msg.aws_template_failed
```

Comment:
In this case we should create a common error message in the 'messages.properties'